packages/custom-storage/__tests__/SQLStorage.spec.ts

```diff
@@ -26,7 +26,7 @@ describe('SQLStorage', () => {
 
   it('should check or create tables for storage on init', async () => {
     const expectedQueries = [
-      'CREATE TABLE IF NOT EXISTS mynamespace_public_v1 (id uuid PRIMARY KEY DEFAULT mynamespace_create_uuid(), name text NOT NULL, description text, thumbnail text, private boolean, config json, last_modified timestamp NOT NULL DEFAULT now());',
+      'CREATE TABLE IF NOT EXISTS mynamespace_public_v1 (id uuid PRIMARY KEY DEFAULT mynamespace_create_uuid(), name text NOT NULL, description text, thumbnail text, isprivate boolean, config json, lastmodified timestamp NOT NULL DEFAULT now());',
```

Comment:
I now about the private 'reserved' word issue, but out of curiosity, why did you change 'last_modified'?

Reply:
This was a quick fix for a mismatch between `lastModified` in object and `last_modified` in the DB; to avoid a deeper change when we wanted to finish v0.2 soon.
But thinking on it again, it could be better to create another field in columns for `nameDB` or just create an object adapter to transform between JS and DB.

Reply:
Feel free to improve the 'FIELDS' with any mechanism you think is better to avoid mistakes